app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1038,61 +1038,37 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     )
 
     /**
-     * Stores the number of installed add-ons for telemetry purposes
+     * Storing number of installed add-ons for telemetry purposes
```

Comment:
I am not quite sure about the tense between "Storing" over "Stores". I personally prefer using "Stores" here.

Reply:
hmm, not sure why this is part of the revert.  I'll double check

Reply:
Confirmed that this is part of the credit card telemetry refactor change.  Just to ensure that I am reverting everything, I'll keep this as part of the revert.